atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceRuntimeConfig.java

```diff
@@ -94,6 +99,11 @@ public Integer sweepReadThreads() {
         return AtlasDbConstants.DEFAULT_SWEEP_CASSANDRA_READ_THREADS;
     }
 
+    @Value.Default
+    public int poolSize() {
```

Comment:
I don't think we want this to be live-reloadable?

Reply:
`concurrentGetRangesThreadPoolSize` reads it in `poolSize() * servers().numberOfThriftHosts()`. 

should `servers` be reloadable but not `poolSize` then? (I dunno how weird that is)

Reply:
yeah, afraid this one should probably stay in install: we don't actually allow the size of the connection pools to be adjusted at runtime I think, so having this here is a bit misleading.

ugh, the multiplication logic should live in the Reloadable one then.

Reply:
then we also have to remove `concurrentGetRangesThreadPoolSize` from runtimeconfig since it needs `poolSize` to compute

Reply:
unless we set poolsize to a default for the multiplication.

Reply:
> we don't actually allow the size of the connection pools to be adjusted at runtime I think

We also don't let `concurrentGetRangesThreadPoolSize` be adjusted as runtime. It's fetched once when creating the `TransactionManager`.